src/App.scss

```diff
@@ -1,3 +1,24 @@
+@import './src/common/styles/variables.scss';
+
 .App {
   text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.active {
+  font-weight: bold;
+  font-family: 'Amiri', serif;
+}
+
+.breadcrumbsWrapper {
```

Comment:
It's better to create stylesheet for single component, not general stylesheet

Reply:
Improved, it's just class used in example, so I've moved it and added comment to remove it later (when remove example) :)